src/main/resources/com/fulcrumgenomics/umi/CollectDuplexSeqMetrics.R

```diff
@@ -0,0 +1,118 @@
+#################################################################################
+# The MIT License                                                               #
+# Copyright (c) 2017 Fulcrum Genomics LLC                                       #
+# Permission is hereby granted, free of charge, to any person obtaining a copy  #
+# of this software and associated documentation files (the "Software"), to deal #
+# in the Software without restriction, including without limitation the rights  #
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell     #
+# copies of the Software, and to permit persons to whom the Software is         #
+# furnished to do so, subject to the following conditions:                      #
+# The above copyright notice and this permission notice shall be included in    #
+# all copies or substantial portions of the Software.                           #
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR    #
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,      #
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE   #
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER        #
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, #
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN     #
+# THE SOFTWARE.                                                                 #
+#################################################################################
+
+#################################################################################
+# R script to generate QC plots for duplex sequencing data QC's with
+# CollectDuplexSeqMetrics tools.  All arguments are positional and required:
+#   1. The family size file with data on families by start/stop, ss and ds
+#   2. The duplex family size file
+#   3. The duplex yield metrics file
+#   4. The UMI metrics file
+#   5. An output PDF to write
+#   6+ One or more strings to use in plot titles to ID the sample/dataset
```

Comment:
The plus was intentional to indicate arguments 6 and on.  I've added a period after the plus to make it more clear it's not just a mistake.